src/app/components/community/sidebar/sidebar.ts

```diff
@@ -49,18 +49,37 @@ export default class AppCommunitySidebar extends Vue {
 	@Prop(Array)
 	collaborators!: User[];
 
-	@Prop(Boolean)
-	hasMoreCollaborators!: boolean;
+	@Prop(Number)
+	collaboratorCount!: number;
 
 	@Prop(Number)
 	initialCollaboratorCount!: number;
 
 	@State
 	app!: Store['app'];
 
-	collaboratorListCollapsed = false;
+	currentCollaborators: User[] = [];
+	currentCollaboratorCount = 0;
+
+	collaboratorListCollapsed = true;
 	isLoadingMoreCollaborators = false;
 	isShowingShare = false;
+	loadedAll = false;
```

Comment:
I don't understand what this does. It doesn't get set to `true` ever.

Reply:
gud catch, should've set to true when loading

Reply:
Also renamed to be clearer what this is loading since sidebar does multiple things